client/js/Presenter.js

```diff
@@ -77,8 +77,22 @@ var Presenter = {
 
     episodeID = element.getAttribute("episode")
     seriesID = element.getAttribute("series")
+    
+    possibleLiveURL = element.getAttribute("liveURL")
 
     switch(view) {
+	  case "live":
+        resourceLoader.loadResource(resourceLoader.BASEURL + "templates/Live.xml.js",
+          episodes,
```

Comment:
As the call to `Episode.live()` was removed in 500671 and this code doesn't run in a callback anymore, the `episodes` variable is undefined here. Since `resourceLoader.loadResource()` expects a `data` value here, I'd just pass `null` here.